wallet/src/de/schildbach/wallet/ui/LockScreenActivity.kt

```diff
@@ -120,8 +124,26 @@ class LockScreenActivity : SendCoinsQrActivity() {
     }
 
     private fun hasNavBar(): Boolean {
+        val tm: TelephonyManager = getSystemService(Context.TELEPHONY_SERVICE) as TelephonyManager
+        // emulator
+        if ("Android" == tm.networkOperatorName || Build.FINGERPRINT.startsWith("generic")) {
+            return true
+        }
         val id: Int = resources.getIdentifier("config_showNavigationBar", "bool", "android")
-        return id > 0 && resources.getBoolean(id)
+
+        // Krip devices seem to incorrectly report config_showNavigationBar
+        val isKripDeviceWithoutNavBar = Build.BRAND == "KRIP" && when (Build.MODEL) {
+            "K5", "K5c", "K5b", "K4m", "KRIP_K4", "K6" -> true
+            else -> false
+        }
```

Comment:
These models were chosen based on device images from Google Play that seem to clearly show that these devices have hardware buttons for back, etc.

Only the K5 was tested.

Reply:
@HashEngineering,
this change broke it on my K600A device (it doesn't have the physical buttons):
![image](https://user-images.githubusercontent.com/8437401/91730454-2f5ddb00-eba6-11ea-8501-137833edd0e6.png)

Reply:
oh cool, glad you have the device to check it!

Reply:
Fixed here: https://github.com/dashevo/dash-wallet/pull/491/commits/8c81b4b4cf65ee4c226e06e6355d9d8e52f4077a

Reply:
it works 👍